src/components/SectionName.jsx

```diff
@@ -0,0 +1,208 @@
+import { InfoIcon } from '@chakra-ui/icons';
+import {
+  Button,
+  Container,
+  Flex,
+  GridItem,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Select,
+  SimpleGrid,
+  Spacer,
+  Text,
+  Tooltip,
+  VStack,
+} from '@chakra-ui/react';
+import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import './SectionName.css';
+
+function SectionName() {
+  const [startDate, setDate] = useState(new Date());
+  const [startTime, setStartTime] = useState(true);
+  const handleStartTimeClick = () => setStartTime(!startTime);
+  const [endTime, setEndTime] = useState(true);
+  const handleEndTimeClick = () => setEndTime(!endTime);
+
+  const user = {
+    segments: [
+      { id: '0', name: 'segment0' },
+      { id: '1', name: 'segment1' },
+    ],
+  };
+
+  const onChange = date => setDate(date);
+
+  return (
+    <div>
+      <Container maxW="100vw">
+        <VStack spacing="23px" align="left">
+          <Text fontWeight="600" fontSize="2xl">
+            Section Name
+          </Text>
+          <SimpleGrid columns={4} rows={3} spacingX="64px" spacingY="68px">
+            <GridItem colSpan={1} rowSpan={1} width="200px">
+              <VStack spacing="8px" align="left">
+                <Text fontWeight="500" fontSize="md">
+                  Survey Segment
+                </Text>
+                <Select>
+                  {user.segments.map(segment => (
+                    <option value="" key="">
+                      {segment.name}
+                    </option>
+                  ))}
+                </Select>
+              </VStack>
+            </GridItem>
+            <GridItem colSpan={1} rowSpan={1} width="200px">
+              <Text fontWeight="500" fontSize="md">
+                Date (MM/DD/YYYY)
+              </Text>
+              <DatePicker selected={startDate} onChange={onChange} />
```

Comment:
You can just pass in the arrow function directly into the `onChange` prop.

`onChange={(e) => setSomeState(e)}`